### PR TITLE
1ES PT Template Migration and Resolving Credscans (Supression)

### PR DESCRIPTION
**Pull request by v-pchippalat**
#### Details

Migrating the standard YAML pipeline to 1ES PT .
##### Motivation

Address the non-compliance of having the standard build pipelines.

### CODE REVIEW

File: azure-pipeline/build-artifacts-and-run-tests-job.yaml [hunk 1]
```diff
@@ -61,12 +61,12 @@ steps:
       inputs:
           BuildDropPath: '$(Build.ArtifactStagingDirectory)'
       displayName: 'SBOM Generation Task'
-
-    - task: PublishBuildArtifacts@1
-      inputs:
-          pathtoPublish: '$(Build.ArtifactStagingDirectory)'
-          artifactName: drop
-      displayName: 'Publish build artifacts'
+    # Moved this Task to the Main file as OutPut as per 1ES PT Guidelines
```
Comment (lamaks) [inline azure-pipeline/build-artifacts-and-run-tests-job.yaml:64]: Just remove if not needed

File: azure-pipeline/Accessibility_Insights_Service_CI_1ESPT.yml
Comment (lamaks) [inline azure-pipeline/Accessibility_Insights_Service_CI_1ESPT.yml:1]: Delete https://github.com/microsoft/accessibility-insights-service/blob/main/Accessibility_Insights_Service_CI_Yml.yml

File: azure-pipeline/build-artifacts-and-run-tests-job.yaml (hunk 1 above)
Comment (v-pchippalat) [reply azure-pipeline/build-artifacts-and-run-tests-job.yaml]: Deleted the commented Code.

File: azure-pipeline/Accessibility_Insights_Service_CI_1ESPT.yml
Comment (v-pchippalat) [reply azure-pipeline/Accessibility_Insights_Service_CI_1ESPT.yml:1]: Removed this duplicate file

**Review by lamaks — APPROVED** [review]

**Pull request merged**